CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Model/Handbook.swift

```diff
@@ -14,11 +14,11 @@ struct Handbook: Identifiable {
     var text: String
     var logo: String
     var image: String
-    var color1: Color
-    var color2: Color
+    var colorTop: Color
```

Comment:
If you want to write such model to the Database, it will better not to mix SwiftUI and Foundation types

Reply:
Ah ok interesting. Just looked this up online, they saved the RGB values. I'll keep that in mind. Thank you

Reply:
Yep, RGB is ok